util/unittests/test_process.py

```diff
@@ -116,7 +116,7 @@ def test01_getpid_listeningonport(self):
                          'Testing the returned PID. Local')
 
 
-    def test02_getpid_listeningonport(self):
+    def test_getpid_listeningonport(self):

```

Comment:
__[flake8]__
* [F811] redefinition of unused 'test_getpid_listeningonport' from line 108

<!-- Comment by [SideCI](https://sideci.com) -->